### ISSUE TITLE
tor error

### ISSUE THREAD

**Issue by taeasy** [openwrt/packages#1147]
make[4]: Entering directory `/home/main/x86/build_dir/target-mips_34kc_uClibc-0.9.33.2/tor-0.2.5.10'
depbase=`echo src/common/sandbox.o | sed 's|[^/]*$|.deps/&|;s|\.o$||'`;\
	mips-openwrt-linux-uclibc-gcc -DHAVE_CONFIG_H -I.  -I./src/ext -Isrc/ext -I./src/common -Isrc/common -I./src/or -Isrc/or -DSHARE_DATADIR="\"/usr/share\"" -DLOCALSTATEDIR="\"/var\"" -DBINDIR="\"/usr/bin\"" -I/home/main/x86/staging_dir/target-mips_34kc_uClibc-0.9.33.2/usr/include -I/home/main/x86/staging_dir/target-mips_34kc_uClibc-0.9.33.2/include -I/home/main/x86/staging_dir/toolchain-mips_34kc_gcc-4.8-linaro_uClibc-0.9.33.2/usr/include -I/home/main/x86/staging_dir/toolchain-mips_34kc_gcc-4.8-linaro_uClibc-0.9.33.2/include  -I./src/common -I/home/main/x86/staging_dir/target-mips_34kc_uClibc-0.9.33.2/usr/include -I/home/main/x86/staging_dir/target-mips_34kc_uClibc-0.9.33.2/usr/include   -Os -pipe -mno-branch-likely -mips32r2 -mtune=34kc -fno-caller-saves -fhonour-copts -Wno-error=unused-but-set-variable -msoft-float -mips16 -minterlink-mips16 -MT src/common/sandbox.o -MD -MP -MF $depbase.Tpo -c -o src/common/sandbox.o src/common/sandbox.c &&\
	mv -f $depbase.Tpo $depbase.Po
src/common/sandbox.c: In function 'sb_mmap2':
src/common/sandbox.c:345:58: error: 'MAP_STACK' undeclared (first use in this function)
        SCMP_CMP(3, SCMP_CMP_EQ,MAP_PRIVATE|MAP_ANONYMOUS|MAP_STACK));
                                                          ^
src/common/sandbox.c:205:47: note: in definition of macro 'seccomp_rule_add_2'
   seccomp_rule_add((ctx),(act),(call),2,(f1),(f2))
                                               ^
src/common/sandbox.c:345:8: note: in expansion of macro 'SCMP_CMP'
        SCMP_CMP(3, SCMP_CMP_EQ,MAP_PRIVATE|MAP_ANONYMOUS|MAP_STACK));
        ^
src/common/sandbox.c:345:58: note: each undeclared identifier is reported only once for each function it appears in
        SCMP_CMP(3, SCMP_CMP_EQ,MAP_PRIVATE|MAP_ANONYMOUS|MAP_STACK));
                                                          ^
src/common/sandbox.c:205:47: note: in definition of macro 'seccomp_rule_add_2'
   seccomp_rule_add((ctx),(act),(call),2,(f1),(f2))
                                               ^
src/common/sandbox.c:345:8: note: in expansion of macro 'SCMP_CMP'
        SCMP_CMP(3, SCMP_CMP_EQ,MAP_PRIVATE|MAP_ANONYMOUS|MAP_STACK));
        ^
src/common/sandbox.c: In function 'sigsys_debugging':
src/common/sandbox.c:1643:35: error: 'mcontext_t' has no member named 'M_SYSCALL'
   syscall = (int) ctx->uc_mcontext.M_SYSCALL;
                                   ^
make[4]: *** [src/common/sandbox.o] Error 1
make[4]: Leaving directory `/home/main/x86/build_dir/target-mips_34kc_uClibc-0.9.33.2/tor-0.2.5.10'
make[3]: *** [all] Error 2
make[3]: Leaving directory `/home/main/x86/build_dir/target-mips_34kc_uClibc-0.9.33.2/tor-0.2.5.10'
make[2]: *** [/home/main/x86/build_dir/target-mips_34kc_uClibc-0.9.33.2/tor-0.2.5.10/.built] Error 2
make[2]: Leaving directory `/home/main/x86/feeds/packages/net/tor'
make[1]: *** [package/feeds/packages/tor/compile] Error 2
make[1]: Leaving directory `/home/main/x86'
make: *** [package/tor/compile] Error 2
main@bogon:~/x86$

**Comment by hnyman** [comment]
Duplicate of #1161 and #1097
